wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/DeviceDetailsBottomView.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+
+struct DeviceDetailsBottomView: View {
+    @Binding var viewModel: DeviceInfoViewModel
+    var body: some View {
+        Text("self.settings.device_details.fingerprint.subtitle".localized)
```

Comment:
can we try to have smaller meaningful parts in Body

like :

```
title
resetSessionButton
subtitle
...
```

I think that would simplify readability